### DISCUSSION TITLE
i have problem with torch.compile()

### DISCUSSION THREAD

**Discussion by Sariskiattitha** [mrdbourke/pytorch-deep-learning#760] [Q&A]
I've come across the PyTorch 2.0 lecture by MrDBourke, and it's really interesting to learn how to make the model train very fast.

Then, I have two questions:

1.) I use torch.compile(model) in the forward step, and it returns an error. Has anyone else come across this problem?

This is the error.

<img width="1256" alt="Screenshot 2566-12-19 at 16 14 25" src="https://github.com/mrdbourke/pytorch-deep-learning/assets/151040683/6ed5fb97-56ba-40cf-8b2e-a85925492705">

2.) Does anyone know how to allocate more Apple GPU memory to PyTorch? Because in the documentation, torch.mps seems not to have that command, or I really can't find it.

Sincerely, thanks!!

**Comment by shisirkha** [comment]
Hmm, you might go see the actual documentation for this
